docs/proposals/20191030-machine-health-checking.md

```diff
@@ -71,12 +72,12 @@ Enable opt in automated health checking and remediation of unhealthy nodes backe
 - Provide a mechanism to guarantee that application quorum for N members is maintained at any time.
 
 ## Proposal
-The machine health checker (MHC) does a best effort to keep nodes healthy in the cluster, by removing machines backing unhealthy Nodes.
+The machine health checker (MHC) is responsible for marking machines backing unhealthy Nodes with an annotation so that can be remediated by their owning controller.
```

Comment:
Just a nit-pick
...Nodes with an annotation so that **they** can be remediated...